lib/src/main/java/com/diffplug/spotless/kotlin/KtfmtStep.java

```diff
@@ -62,23 +77,36 @@ public static String defaultVersion() {
 		private static final long serialVersionUID = 1L;
 
 		private final String pkg;
+		/**
+		 * Option that allows to apply formatting options to perform a 4 spaces block and continuation indent.
+		 */
+		private final Boolean withDropboxStyle;
 		/** The jar that contains the eclipse formatter. */
 		final JarState jarState;
 
-		State(String version, Provisioner provisioner) throws IOException {
+		State(String version, Provisioner provisioner, Boolean withDropboxStyle) throws IOException {
 			this.pkg = PACKAGE;
+			this.withDropboxStyle = withDropboxStyle;
 			this.jarState = JarState.from(MAVEN_COORDINATE + version, provisioner);
 		}
 
 		FormatterFunc createFormat() throws Exception {
 			ClassLoader classLoader = jarState.getClassLoader();
-
 			Class<?> formatterClazz = classLoader.loadClass(pkg + ".ktfmt.FormatterKt");
-			Method formatterMethod = formatterClazz.getMethod(FORMATTER_METHOD, String.class);
-
 			return input -> {
 				try {
-					return (String) formatterMethod.invoke(formatterClazz, input);
+					if (withDropboxStyle) {
+						Class<?> formattingOptionsClazz = classLoader.loadClass(pkg + ".ktfmt.FormattingOptions");
+						Object formattingOptions = formattingOptionsClazz.getConstructor(
+								int.class, int.class, int.class).newInstance(
+										MAX_WIDTH_LINE, BLOCK_INDENT, CONTINUATION_INDENT);
+						Method formatterMethod = formatterClazz.getMethod(FORMATTER_METHOD, formattingOptionsClazz,
```

Comment:
This way of triggering `--dropbox-style` is fragile.  Having watched several formatters with "no options" evolve over years, I am quite confident that this code will break someday.

If you want this PR to fix your `--dropbox-style` problem forever, I recommend that you first get `ktfmt 0.16` to add a method `static FormattingOptions.dropboxStyle()`.  Right now, `ktfmt` doesn't think it has a public API for the dropbox settings, but it accidentally has one, and it will accidentally break someday.

If you can't get that merged into `ktfmt` / in the meantime, I am okay with merging this as-is, but please add this comment:

```java
// we are duplicating the result of this parsing logic from ktfmt 0.15
// https://github.com/facebookincubator/ktfmt/blob/59f7ad8d1fde08f3402a013571c9997316083ebf/core/src/main/java/com/facebook/ktfmt/ParsedArgs.kt#L37
// if the code above changes in a future version, we will need to change this code
```

Reply:
I will make the suggestion to ktfmt and add this comment for now then.